core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -601,6 +601,10 @@ private static ElevationProvider createElevationProvider(GraphHopperConfig ghCon
                 ? ghConfig.getBool("graph.elevation.calcmean", false)
                 : ghConfig.getBool("graph.elevation.calc_mean", false);
 
+        boolean interpolate = ghConfig.has("graph.elevation.interpolate")
+                ? ghConfig.getBool("graph.elevation.interpolate", false)
+                : eleCalcMean;
```

Comment:
Let's remove the fall back for graph.elevation.calcmean and throw an error if it is still specified.